x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -356,18 +374,32 @@ export class FleetPlugin
               asInternalUser: service.asInternalUser,
             };
           },
-          authz: await getAuthzFromRequest(request),
+          authz,
           epm: {
             // Use a lazy getter to avoid constructing this client when not used by a request handler
             get internalSoClient() {
-              return appContextService
-                .getSavedObjects()
-                .getScopedClient(request, { excludedExtensions: [SECURITY_EXTENSION_ID] });
+              return getInternalSoClient();
             },
           },
           get spaceId() {
             return deps.spaces.spacesService.getSpaceId(request);
           },
+
+          async getSoClient() {
```

Comment:
it seems weird to me that method return both a saved object client and some limited packages, should we get two property on the fleet context?

````
get soClient()
get limitedToPackages()
```

Reply:
@ashokaditya - can you make this change tomorrow?

Reply:
done 2886e3dc4a2f459c0e515537c5a01fd2556c96fb